autolab/Autolab
# Lint views/autograders, fix help-block gap from input

## Summary

### Summary generated by Reviewpad on 06 Oct 23 17:28 UTC
This pull request includes two patches. 

The first patch makes changes to the `app/views/autograders/_form.html.erb` file. It updates some JavaScript code and fixes the gap in the help block from input elements. It also reorganizes the code for better readability and maintains consistent indentation. The changes include 36 insertions and 36 deletions.

The second patch updates the form path in the same file. It modifies the `form_for` statement to use the `course_assessment_autograder_path` helper and directly specifies the URL for the form. This change includes 3 insertions and 3 deletions.


## Description

- lint `views/autograders` using `bundle exec erblint app/views/autograders/*.html.erb -a`
- Edit `views/autograders/_form.html.erb` so that the help block under "VM Image" doesn't have a lot of vertical white space
- Change any links to use "_path()" format instead of bracket format

## Motivation and Context


- as part of summer linting

## How Has This Been Tested?



- Go to Assessment with autograder
- See that other than help block change, visually the "Autograder Settings" looks the same
- Ensure the buttons still work (uploading files, save settings, delete)

Before:

<img width="1504" alt="Screenshot 2023-08-19 at 9 43 54 PM" src="https://github.com/autolab/Autolab/assets/25730111/af1c782b-dc7e-4344-9cb8-f9a9dc15fe5e">

After:

<img width="1512" alt="Screenshot 2023-08-19 at 9 43 02 PM" src="https://github.com/autolab/Autolab/assets/25730111/d7e4ef7d-c7e8-480f-9a12-23716c611528">

## Checklist:
